Resources/Prototypes/Recipes/Crafting/Graphs/improvised/flowercrown.yml

```diff
@@ -0,0 +1,16 @@
+- type: constructionGraph
+  id: flowercrown
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: flowercrown
+          steps:
+            - prototype: FoodPoppy
```

Comment:
For this construction step, do not use the prototype step. You'll want to either create a tag or a material for flowers in general. This also means that you could expand this to work with different flowers. You can read more about construction here: [https://docs.spacestation14.io/en/content/construction/index](url)

Reply:
Added "Flower" tag, converted to use Flower.